share/scripts/inverse/run_gromacs.sh

```diff
@@ -28,6 +28,27 @@ EOF
    exit 0
 fi
 
+tasks=$(get_number_tasks)
+
+mdrun_opts="$(csg_get_property --allow-empty cg.inverse.gromacs.mdrun.opts)"
+
+if [[ ${mdrun_opts} == *multidir* ]]; then
+
+  mdirs=$(echo ${mdrun_opts} | grep -o "sim" | wc -l)
```

Comment:
Yep, 'sim' must be part of all the subdirectory names for the '-multidir' option in csg.inverse.gromacs.mdrun.opts. From the user perspective, it would be an error-prone overkill to have yet another option for the number of MPI-tasks =#subdirs < #tasks (overall threads), let alone an extra flag on top of these (I did consider <multidir>yes/no <number>#</number></multidir>). The way I did it the user only needs to specify (or not) the option itself and nothing else to support it. 

I don't get how having conf_sim.gro in the directory can break things down?